### PR TITLE
site/guide: Small nits on the external auth guide

### PR DESCRIPTION
**Pull request by stevesloka**
Signed-off-by: Steve Sloka <slokas@vmware.com>

### CODE REVIEW

File: site/_guides/external-authorization.md [hunk 1]
```diff
@@ -36,7 +34,7 @@ the request to the upstream application.
 It is helpful to have a mental model of how requests flow through the various
 servers involved in authorizing HTTP requests.
 The flow diagram below shows the actors that participate in the successful
-authorization of a HTTP request.
+authorization of an HTTP request.
```
Comment (skriss) [inline site/_guides/external-authorization.md:37]: is this a US vs AUS thing? 😂

**Review by skriss — APPROVED** [review]
Looks fine to me

File: site/_guides/external-authorization.md (hunk 1 above)
Comment (stevesloka) [reply site/_guides/external-authorization.md:37]: I dunno, I just copy into google docs and let it find the issues. 🤷

Comment (jpeach) [reply site/_guides/external-authorization.md:37]: "a HTTP request" is correct IMHO, but I suppose either could be acceptable

File: site/_guides/external-authorization.md [hunk 2]
```diff
@@ -3,8 +3,6 @@ title: External Authorization Support
 layout: page
 ---
 
-<div id="toc" class="navigation"></div>
```
Comment (jpeach) [inline site/_guides/external-authorization.md:6]: Why remove this?

I can see that it doesn't display correctly on the published site, but it did work locally. This doc is long enough that it needs this index.

File: site/_guides/external-authorization.md [hunk 3]
```diff
@@ -18,7 +16,7 @@ that implements this protocol.
 
 You can bind an authorization server to Contour by creating a
 [`ExtensionService`][4] resource.
-This resource tells Contour that the service exists, and that it should
+This resource tells Contour the service exists, and that it should
```
Comment (jpeach) [inline site/_guides/external-authorization.md:19]: "that" is clearer, though this is ok too

File: site/_guides/external-authorization.md [hunk 4]
```diff
@@ -85,19 +83,19 @@ server to verify or obtain an authorization token.
...
-responded to the client with a HTTP error.
+responded to the client with an HTTP error.
 
 ## Using the Contour Authorization Server
 
 The Contour project has built a simple authorization server named
 [`contour-authserver`][1]. `contour-authserver` supports an authorization
-testing server and a HTTP basic authorization server that accesses
-credentials in stored in [htpasswd][2] format.
+testing server, and an HTTP basic authorization server that accesses
```
Comment (jpeach) [inline site/_guides/external-authorization.md:97]: ugh, Oxford comma :(

**Review by jpeach — APPROVED** [review]
We should restore (and fix the TOC), but otherwise 👍

File: site/_guides/external-authorization.md (hunk 1 above)
Comment (skriss) [reply site/_guides/external-authorization.md:37]: heh, "a HTTP request" sounds wrong to my ear, but whatever.

File: site/_guides/external-authorization.md (hunk 2 above)
Comment (stevesloka) [reply site/_guides/external-authorization.md:6]: There's no index, just a box that gets rendered that's empty: 
image

Comment (jpeach) [reply site/_guides/external-authorization.md:6]: Screen Shot 2020-10-07 at 8 11 01 am

Locally it looks like this. There's some sort of javascript problem on the published site.

Comment (jpeach) [reply site/_guides/external-authorization.md:6]: We can remove the TOC here and file a follow-up issue.

**Comment by stevesloka** [comment]
> We should restore (and fix the TOC)

Yup 100%! I didn't know that it didn't render when I removed, just looked like a mistake on the published site. I'll open an issue and see if we can get this fixed up. I do agree that this is super helpful to have in place.

**Pull request merged**